pebblo/app/config/config.yaml

```diff
@@ -7,3 +7,5 @@ reports:
   format: pdf
   renderer: xhtml2pdf
   outputDir: ~/.pebblo
+classifier:
+  anonymize_all_entities: True
```

Comment:
In YAML, the existing fields follow the camel case convention, the recently added one, anonymize_all_entities, uses a snake case, which looks odd.

Reply:
Updated